qa/mixed-cluster/src/test/java/org/elasticsearch/backwards/IndexingIT.java

```diff
@@ -309,9 +312,15 @@ public void testSyncedFlushTransition() throws Exception {
         try (RestClient newNodeClient = buildClient(restClientSettings(),
             nodes.getNewNodes().stream().map(Node::getPublishAddress).toArray(HttpHost[]::new))) {
             Request request = new Request("POST", index + "/_flush/synced");
-            List<String> warningMsg = List.of("Synced flush was removed and a normal flush was performed instead. " +
-                "This transition will be removed in a future version.");
-            request.setOptions(RequestOptions.DEFAULT.toBuilder().setWarningsHandler(warnings -> warnings.equals(warningMsg) == false));
+            final String v7MediaType = XContentType.VND_JSON.toParsedMediaType()
```

Comment:
I am not sure we would like to use synced flush here. Since it was deprecated in 7.x and is only available under rest compatibility in 8 I guess we should just use flush
WDYT?

Reply:
The whole test is called `testSyncedFlushTransition` and starts with `assertTrue("bwc version is on 7.x", nodes.getBWCVersion().before(Version.V_8_0_0));` so I think it's right to still use synced-flush here. We could reasonably duplicate this test as one which just uses a regular flush and will continue to work once 7.x is no more.